docs/real-world-examples/pneumonia.md

```diff
@@ -0,0 +1,54 @@
+
+We mimic a real-world scenario where 3 hospitals in 3 different regions want to collaborate on training a model to detect pneumonia from chest radiographs. The hospitals have their own data, and they want to train a model on all data without directly sharing data with each other, or with a central entity. The model will be trained in a federated manner, where each hospital will train a model on its own data, and the models will be aggregated to produce a final model.
+
+> For the sake of simplicity, we will only provision an _open_ setup. Do not upload sensitive data to it! 
+
+## Prerequisites
+To enjoy this tutorial, you will need to:
+- have an active [Azure subscription](https://azure.microsoft.com) that you can use for development purposes,
+- have permissions to create resources, set permissions, and create identities in this subscription (or at least in one resource group),
+  - Note that to set permissions, you typically need _Owner_ role in the subscription or resource group - _Contributor_ role is not enough. This is key for being able to _secure_ the setup.
+- [install the Azure CLI](https://learn.microsoft.com/en-us/cli/azure/install-azure-cli),
+- have a way to run bash scripts;
+- **fork** this repository (_fork_ as opposed to _clone_ because you will need to create GitHub secrets and run GitHub actions to prepare the data).
+
+## Procedure
+The procedure to run this example  has three phases:
+- provision the Azure resources;
+- prepare the data;
+- run the job.
+
+Each phase is described in the subsections below.
+
+### Provisioning
+This phase is easy. Just follow the instructions in the [quickstart](../quickstart.md) to provision an open sandbox. Make note of the name of the resource group you provisioned, as well as the name of the workspace.
+
+### Data preparation
+The goal of this phase is to upload to Azure ML 3 distinct datasets (one per silo), each subdivided into train, test, and validation. This is achieved by running a GitHub action that will download the dataset from [Kaggle](https://www.kaggle.com/datasets/paultimothymooney/chest-xray-pneumonia), split it evenly between the silos, and upload the 3 data assets. We will need to first create an Azure Service Principal with enough permissions to create the data assets in Azure ML. The whole procedure is explained below, and closely follows [the one](https://github.com/Azure/medical-imaging/blob/main/federated-learning/README.md#1-prepare-your-experiment) developed by Harmke Alkemade _et al_ .
+
+1. Create a service principal with contributor access to your resource group by running the following Azure CLI command.
```

Comment:
I would rather provide a job in yaml that users can run in their own workspace, it would completely save the step to create a fork, then an SP, etc.

Basically, you could provide something like this:
https://github.com/Azure/azureml-assets/blob/main/benchmark/src/jobs/create_places2_dataset.yml

and then just ask users to run `az ml job create` with that job first, it would unpack the dataset on each silo

Reply:
I doubt this is going to be much easier since we're also going to have to deal with some sort of secrets for the Kaggle credentials, but sure I'll try to use an Azure ML command job instead